CHANGELOG.md

```diff
@@ -16,12 +16,16 @@ https://github.com/bitcrowd/sshkit.ex/compare/v0.0.2...HEAD
 
 ### New features:
 
+* Added support for uploading/downloading files [#32]
```

Comment:
I'd suggest to add `SSHKit` here, since it was already possible on other modules...